static/katex.less

```diff
@@ -301,6 +301,26 @@
         }
     }
 
+    // Set a min-height for some horizontal lines so their display
+    // will show at least one screen pixel.
+    @media screen {
+        .mfrac .frac-line,
+        .overline .overline-line,
+        .underline .underline-line {
+            min-height: 1px;
+        }
+    }
+
+    // Modify min-height for retina displays. Don't force two screen pixels.
+    @media screen and (-webkit-min-device-pixel-ratio: 2.0),
+        screen and (min-resolution: 192dpi) {
```

Comment:
I'm going to merge this, but I'm curious about the choice of dpi... is it b/c Window's default 1x resolution is 96dpi?

Reply:
It's not just a Windows thing. According to [this post](https://www.w3.org/blog/CSS/2012/06/14/unprefix-webkit-device-pixel-ratio/),192 dpi and 2 dppx are the same thing. Now, I know that a blog post from W3C is not as authoritative as a W3C standard, but I still take it as pretty good information.

And by using dpi, we get support from IE, which does not support dppx.

Reply:
Interesting.  Hmm... I wonder if we can we can supply both `dppx` values and `dpi` values for `min-resolution` in that order.  Not sure if there's a browser out there that supports `dppx` but doesn't do the correct thing with `dpi` as described in the "caveats" section of the blog post.